core/src/main/java/org/apache/accumulo/core/spi/ondemand/DefaultOnDemandTabletUnloader.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.spi.ondemand;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DefaultOnDemandTabletUnloader implements OnDemandTabletUnloader {
+
+  private static final Logger LOG = LoggerFactory.getLogger(DefaultOnDemandTabletUnloader.class);
+  public static final String INACTIVITY_THRESHOLD =
+      "table.custom.ondemand.unloader.inactivity.threshold.seconds";
+  private static final String TEN_MINUTES = Long.toString(MINUTES.toSeconds(10));
+
+  @Override
+  public void evaluate(UnloaderParams params) {
```

Comment:
Could add a simple unit test for this class.  One way to do this would be to have a protected getTime method that the unit test overrides.  If this were done it could be in a follow on PR.

Reply:
Unit test added in 8738077. Had to put it down in the tserver module so that I had access to server side objects.